api/catalog/api/docs/README.md

```diff
@@ -1,6 +1,6 @@
 This documentation is focused towards consumers who are using the Openverse API.
 To contribute to the development of the Openverse API, please refer to the
-[Openverse developer documentation](https://wordpress.githb.io/openverse/).
+[Openverse developer documentation](https://wordpress.github.io/openverse/).
```

Comment:
It looks like this directs to our new documentation site, https://docs.openverse.org/. Per the original issue, I believe we should update this to that instead:
```suggestion
[Openverse developer documentation](https://docs.openverse.org/).
```

Reply:
Hi! thanks for the feedback, i've already corrected and pushed the issue. 

![Captura desde 2023-04-17 18-58-41](https://user-images.githubusercontent.com/105130738/232620062-095211e3-c9f3-414d-bc07-30207ddcd3b3.png)